x-pack/plugins/enterprise_search/public/applications/app_search/components/multi_input_rows/multi_input_rows.test.tsx

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { setMockActions, setMockValues, rerender } from '../../../__mocks__';
+import '../../../__mocks__/shallow_useeffect.mock';
+
+import React from 'react';
+
+import { shallow } from 'enzyme';
+
+import { InputRow } from './input_row';
+
+import { MultiInputRows } from './';
+
+describe('MultiInputRows', () => {
+  const props = {
+    values: ['a', 'b', 'c'],
+    onSubmit: jest.fn(),
+  };
+  const values = {
+    values: ['a', 'b', 'c'],
+    hasEmptyValues: false,
+    hasOnlyOneValue: false,
+  };
+  const actions = {
+    addValue: jest.fn(),
+    editValue: jest.fn(),
+    deleteValue: jest.fn(),
+  };
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+    setMockValues(values);
+    setMockActions(actions);
+  });
+
+  it('renders a InputRow row for each query', () => {
```

Comment:
`value` maybe?

```suggestion
  it('renders a InputRow row for each value', () => {
```

Reply:
Actually, looking through this file it looks like it still mentions `query` quite a bit... like

> it('disables the add button if any query fields are empty', () => {

Rather than trying to update all of the references as review suggestions, you may just want to review this file and make that wording more generic now that it is not curation specific.

Reply:
ha whoops I forgot to find and replace on this file :||

Reply:
https://github.com/elastic/kibana/pull/96881/commits/8d55abc2c6df91296d1fa8330eb30591800003bb

My git history's a bit fucked so I may squash this in with the first commit at some point once I get keyed logic figured out